src/neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -22,6 +24,8 @@ public sealed class PolicyContract : NativeContract
         private const byte Prefix_MaxBlockSize = 12;
         private const byte Prefix_MaxBlockSystemFee = 17;
 
+        private const byte BlockChunkLength = 100;
```

Comment:
That makes for ~2K value, I'd suggest going up to ~8K or 4K at least.

Reply:
https://github.com/neo-project/neo/blob/6886991ca67f2c2148429ba87bf7d782f8cb12c8/src/neo/Ledger/StorageItem.cs#L25 I prefer to have the same length limits, 4096/20 it's around 200.

Reply:
One account one record?

Reply:
That would allow to do the check with one DB access, but DB access is exactly what I'd try to avoid here if possible. I expect this list to not be very big and it's in the hot path for transaction verification, so I expect it to be cached in memory (that's what we have recently implemented).

Although storing one account per DB key doesn't prevent caching these records in any way, so maybe it's not that important at all.

Reply:
Done